eslint/bump.config.yml

```diff
@@ -0,0 +1,5 @@
+files:
+  - filePath: 'package.json'
+    patterns:
+      - '"name"\s*:\s*"\@exadel\/eslint-plugin-esl"[^{}]*"version"\s*:\s*"(\d+\.\d+\.\d+)"'
+      - '"@exadel\/esl\": \"\^(\d+\.\d+\.\d+)\"'
```

Comment:
It appears to be incorrect, as I mentioned earlier. 
The native npm version processes a version without our interventions.